sdk/deviceregistry/Azure.ResourceManager.DeviceRegistry/CHANGELOG.md

```diff
@@ -1,28 +1,14 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
-
-### Breaking Changes
-
-- Updated `Assets` and `AssetEndpointProfiles` resource types to align to Microsoft.DeviceRegistry OpenAPI spec version `2024-09-01-preview`.
+## 1.0.0-beta.2 (Unreleased)
 
 ### Features Added
 
 - Added `SchemaRegistries`, `SchemaRegistries/Schemas` and `SchemaRegistries/Schemas/SchemaVersions` resource types from Microsoft.DeviceRegistry OpenAPI spec version `2024-09-01-preview`.
 - Added `BillingContainers` resource type from Microsoft.DeviceRegistry OpenAPI spec version `2024-09-01-preview`.
-
-## 1.0.0-beta.2 (Unreleased)
-
-### Features Added
-
+- Updated `Assets` and `AssetEndpointProfiles` resource types to align to Microsoft.DeviceRegistry OpenAPI spec version `2024-09-01-preview`.
```

Comment:
@ArthurMa1978, shouldn't this be considered a breaking change from the Assets/AssetEndpointProfiles definitions exposed in the latest published SDK?

Reply:
correct, this is not a breaking change.